R/metadata-helper.R

```diff
@@ -0,0 +1,33 @@
+# Helper fonction to create a metaData variable
+#' @title metaDataHelper
+#' @param data data.frame
+#' @return metaData with default OSP Suite units
+#' @description
+#' Create a metaData object with information about units and dimensions of data
+#' @export
+#' @examples
+#' df <- pkRatioData
+#' pkRatioMetaData <- metaDataHelper(data = df)
+metaDataHelper <- function(data) {

```

Comment:
What is this class? Not sure i understand

Reply:
It is a helper function to create a metaData from a dataset.
I don't know if this can be useful moving forward. The idea was to be able to generate easily the metaData of your dataset if you don't have it.
The @examples is  what caused the crash of the build, I am committing the fix now.